modules/admin/admin-ui/src/main/resources/web/admin/common/js/util/htmlarea/dialog/CharMapDialog.ts

```diff
@@ -1,10 +1,11 @@
 module api.util.htmlarea.dialog {
 
     import i18n = api.util.i18n;
+    import editor = CKEDITOR.editor;
 
```

Comment:
Change this to ``import HTMLAreaEditor = CKEditor.editor;`` (in all classes)-
It's a bit confusing when it says "``(editor: editor)``" in function signature.